tests/comfy_cli/command/test_run_json.py

```
@@ -0,0 +1,1581 @@
+
+
+def _run_execute_capture(workflow_path, capsys, **overrides):
+    """Run execute() and return the parsed JSON events from stdout."""
+    kwargs = dict(
+        host="127.0.0.1",
+        port=8188,
+        wait=True,
+        verbose=False,
+        timeout=30,
+        json_mode=True,
+    )
```

Pylint (4.0.5)

[refactor] Consider using '{"host": '127.0.0.1', "port": 8188, "wait": True, "verbose": False, ... }' instead of a call to 'dict'.

(R1735)